guides/disabling-reCAPTCHA.md

```diff
@@ -0,0 +1,58 @@
+# Disabling reCAPTCHA
+
+:::warning SECURITY WARNING
+It is highly reccomended to keep reCAPTCHA enabled, as reCAPTCHA is a security method that can protect your site from some methods of attack!
+Only continue if you intend to re-enable reCAPTCHA after fixing whatever issue you may have!
+:::
+
+## Disabling via .env
+
+In order to disable reCAPTCHA using this method, you need to find and edit your `.env` file, It is usually located at `/var/www/pterodactyl/.env`
+
+:::tip Hidden Files
+In linux, files starting with a `.` are considered to be hidden, so most FTP software may not show it by default. You can see the file by using `ls -a`
+:::
+
+First, lets back up the `.env` file. If you are not already, now is a good time to make sure you are in the `/var/www/pterodactyl` directory
+
+```bash
+cp .env .env.bkp
+```
+
+You can now edit the `.env` file using your favorite editor to set the fields below
+
+```bash
+# Ignores database settings
+APP_ENVIRONMENT_ONLY=true
+
+# Disable reCAPTCHA. You may need to add this to the end of your file.
```

Comment:
I'd emphasize more strongly that if you do not have this, then you must add it to the end of your file. I've seen complaints in Discord about them not having such a line in their file because it's easy to miss